js/src/api/pubsub/parity/parity.js

```diff
@@ -185,14 +185,7 @@ export default class Parity extends PubsubBase {
     return this.addListener(this._api, 'parity_localTransactions', (error, transactions) => {
       error
         ? callback(error)
-        : callback(null, transactions => {
-          Object.values(transactions)
-            .filter(tx => tx.transaction)
-            .map(tx => {
-              tx.transaction = outTransaction(tx.transaction);
-            });
-          return transactions;
-        });
+        : callback(null, transactions);
```

Comment:
Why do we get rid of the `outTransaction` here?

Reply:
Good catch. Brought it back.